client/src/components/users/InvitationsListModal.vue

```diff
@@ -0,0 +1,177 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <div class="invitations-list-container">
+    <div class="invitations-list">
+      <div class="invitations-list-header">
+        <ion-text class="invitations-list-header__title title-h4">
+          {{ $msTranslate('invitationList.title') }}
+        </ion-text>
+        <span class="invitations-list-header__counter body-sm">
+          {{ invitations.length }}
+        </span>
+        <ion-button
+          @click="onInviteClick"
+          class="invitations-list-header__button"
+          v-show="!currentRouteIsUserRoute()"
```

Comment:
This restriction doesn't make much sense at the moment, since the button to add a user is shown on the organization info page and not the users page in small display. Might as well delete it as you did in `InvitationsListPopover.vue`